src/components/Mission/Missions.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-console */
```

Comment:
@IfzaRasool  kindly remove comments as it may reduce the readability of your code and it could increase the time it takes for your application or website to load